ksql-rest-app/src/test/resources/ksql-plan-schema/schema.json

```diff
@@ -0,0 +1,1062 @@
+{
```

Comment:
Currently, the schema includes:

Plan Nodes:
KsqlPlanV1
QueryPlan
PhysicalPlan[Object]
StreamAggregate
DefaultExecutionStepProperties
StreamFilter
StreamFlatMap
StreamGroupBy
StreamGroupByKey
StreamMapValues
StreamSelectKey
StreamSink
StreamSource
WindowedStreamSource
StreamStreamJoin
StreamTableJoin
StreamToTable
StreamWindowedAggregate
TableAggregate
TableFilter
TableGroupBy
TableMapValues
TableSink
TableTableJoin
ExecutionStep

DDL commands:
CreateStreamCommand
KsqlTopic
CreateTableCommand
RegisterTypeCommand
DropSourceCommand
DropTypeCommand

Timestamp Extractors:
MetadataTimestampExtractionPolicy
StringTimestampExtractionPolicy
LongColumnTimestampExtractionPolicy

Formats:
KeyFormat
FormatInfo
WindowInfo

I'm working on follow-ups to clean up the timestamp extractors and format types from this schema.

Reply:
We should drop `KsqlTopic` for sure.